examples/xml-access/xml_access.bal

```diff
@@ -14,25 +14,30 @@ public function main() {
                 <!--Price: $10-->
                 </book>`;
     
-    // You can access child XML items using the field-based or index-based access expressions.
-    io:println(bookXML.author.fname);
-    io:println(bookXML["author"]["fname"]);
+    // You can access child XML items using xml step expressions.
+    io:println(bookXML/<author>/<fname>);
 
-    // Accessing a non-existing child will return `nil`.
-    io:println(bookXML.ISBN.code);
-    io:println(bookXML["ISBN"]["code"]);
-
-    // The result of the above field-based or index-based access is another XML. 
-    // Any XML function can be invoked on top of the resulting XML.
-    io:println(bookXML.author.fname.getTextValue());
-    io:println(bookXML["author"]["fname"].getTextValue());
+    // Accessing a non-existing child will return empty xml sequence.
+    io:println(bookXML/<ISBN>/<code>);
 
     // You can also retrieve attributes of the resulting child XML.
-    io:println(bookXML.author.fname@["title"]);
-    io:println(bookXML["author"]["fname"]@["title"]);
+    io:println(bookXML/<author>/<fname>.title);
+
+    // You can match descendent elements using following steping access syntax.
+    io:println(bookXML/**/<fname>);
+
+    // Select all children using bellow syntax.
```

Comment:
`// Use the following syntax to select all the child elements`
or
`// Use the following syntax to select all the children`

Reply:
children will include non elements (pi, comment, text)